staubli_tx90_gazebo/launch/tx90_gazebo.launch

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="paused" default="false"/>
+  <arg name="gui" default="true"/>
+  <arg name="remap" default="false"/>
```

Comment:
Just wondering: should we perhaps rename this argument to something like `rosi_compat` or `rosind_compat`?